README.md

```diff
@@ -116,7 +120,23 @@ Various inputs are defined in [`action.yml`](action.yml) to let you configure th
 | `repo-token` | Token to use to authorize label changes. Typically the GITHUB_TOKEN secret, with `contents:read` and `pull-requests:write` access | N/A |
 | `configuration-path` | The path to the label configuration file | `.github/labeler.yml` |
 | `sync-labels` | Whether or not to remove labels when matching files are reverted or no longer changed by the PR | `false`
+| `dot` | Whether or not to auto-include paths starting with dot (e.g. `.github`) | `false`
```

Comment:
WDYT about dropping the configuration option and making `dot` always true. 

If someone _really_ wants to ignore dot files, that is another glob expression, right? (Something already supported.) IMO, if we can solve the same problem with less configuration it will be more user friendly and more likely to "do what I mean".

Reply:
That would be a breaking change. It’s safer to introduce the option, then wait for a few months, then change the default in a major release, then wait again and finally remove the option at all, if necessary.

Reply:
Yes, I agree it changes the behavior, but perhaps a reasonable one. We could also bump the major version to play it safe with expectations. Ultimately, it is not my and it is only a suggestion as an end user.